docs/platforms/android/releases/index.mdx

```diff
@@ -0,0 +1,41 @@
+---
+title: Release Channels
+description: "Learn about the Android SDK's release channels and choose the right one."
+sidebar_order: 7000
+---
+
+The Android SDK uses the following channels to release SDK updates:
+
+## `Pre-release`
+
+Pre-releases (alpha, beta, RC,…) are for larger and potentially more impactful changes, such as new features or major versions.
+
+## `Latest`
+
+We continuously release major/minor/hotfix versions from the `main` branch. These releases go through all our internal quality gates and are very safe to use. They're intended to be the default for most teams.
+
+## `Stable`
+
+Releases are promoted from `Latest` to `Stable` when they've been used in the field for some time and at some scale. The length of time a release has been out, the level of adoption, and other quality and stability metrics are considered. These releases will be indicated on the releases page (https://github.com/getsentry/sentry-cocoa/releases/) with the `Stable` label.
+
+## FAQ
+
+### Where can I find the latest stable release for any given SDK?
+
+On the releases pages of the SDK's GitHub repo. Stable releases will be indicated with the label `(Stable)` in the name of the release. You can also use the search box in the top right to search for the newest releases.
+
+### Does this mean the release channels are not stable?
+
+No. Stability and performance are the top priorities for every release. The `Stable` release status means that the release has been tested and proven to work in the wild and at scale.
+
+### How often do we mark SDKs as stable?
+
+The target cadence is to provide stable releases every month.
+
+### Should I only use stable releases?
+
+This is a personal choice for you, your team, and your company. As stated earlier, every release is considered stable, but `Stable` releases have been tested longer with the tradeoff of a slightly delayed availability. This is also the reason why there isn't a pre-release for every release.
```

Comment:
i think we need to reword this to give the reader/user some level of confidence that stable releases are dependable.  being a personal choice doesn't tell the customer that stable releases are actually stable. can we state anything around having a more consistent experience, fewer issues, and will get patched first if we see any issues?

Reply:
@rahulchhabria wdyt about "As stated earlier, every release is considered stable. On top of that, `Stable` releases have been used in the field for some time and at significant scale, and are the best choice for added dependability and a consistent experience. The tradeoff is a slightly delayed availability."?

For context about "will get patched first": we don't to commit to `Stable` meaning Long Term Support atm, but we will fix recent `Stable` (e.g. current major/minor version, latest `Stable` on previous major) releases on a best effort basis

Reply:
how about 
> `Stable` releases have been tested in the field and at significant scale. They are intended for a dependable and consistent experience. Please note stable releases will get our newest features later than other releases.

Reply:
adopted with minor wording change